vectordraw/templates/xml/points_and_lines.xml

```diff
@@ -0,0 +1,13 @@
+<vertical_demo>
+  <vectordraw url_name="points_and_lines"
+              description="&lt;p&gt;A car is moving in the positive x-direction and is slowing down. Draw the appropriate free-body diagram for this situation. You can create a draggable vector by clicking one of the buttons to the right of the interactive canvas.&lt;/p&gt;&lt;p&gt;&lt;i&gt;Be sure that the &quot;tail&quot; of each vector starts at the center of mass.&lt;/i&gt; Also try to break this - I have error messages for most possible situations.&lt;/p&gt;"
```

Comment:
@bdero 

> Also try to break this - I have error messages for most possible situations.

I know this is in the JSInput version as well, but is it really part of the exercise description? It seems like something that might have been put in when someone was testing/debugging this particular exercise. In that case it might make sense to remove it.